Assets/Scripts/KatamariMovement.cs

```diff
@@ -161,6 +105,27 @@ private void FixedUpdate()
 
         ForceMode forceMode = isGolfHitMode ? ForceMode.Impulse : ForceMode.Force;
 
+        // turn the power bar on and off depending if we can hit hit the ball
+        if (power_bar_active) // changed from powerbaractive?
```

Comment:
I think the comment here is out of date
```suggestion
        if (power_bar_active)
```